quotations/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db.models.fields         import DecimalField
+from django.db.models.fields.related import ForeignKey
+
+from core.models import TimeStampModel
+
+class Quotation(TimeStampModel):
+    application_master = ForeignKey('ApplicationMaster', on_delete='CASCADE')
```

Comment:
제대로 동작 하나요?
아래와 같이 수정해야되지 않나요?
```suggestion
    application_master = ForeignKey('ApplicationMaster', on_delete=models.CASCADE)
```

Reply:
죄송합니다! 수정 중입니다!